### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by zainafzal08**
just moved everything n the repo into "frontend" and added the basic framework of a backend.

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        db_r.execute()
+        return {
+            "msg": "success",
+            "token": t
+        }
+
+@user.route('/user')
+class User(Resource):
+    def get(self):
+        t = request.headers.get('Authorization',None)
```
Comment (DarkPurple141) [inline backend/app.py]: Authorization header so I'm assuming user needs to set creds?

File: backend/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (DarkPurple141) [inline backend/templates/index.html]: Is this to serve the base file?

File: backend/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Click==7.0
+Flask==1.0.2
+itsdangerous==0.24
+Jinja2==2.10
```
Comment (DarkPurple141) [inline backend/requirements.txt]: jinja still here. Is it used?

**Review by DarkPurple141 — APPROVED** [review]
I'm sure all good, haven't run locally. Are you likely to provide a basic explainer for server use?

File: backend/app.py (hunk 1 above)
Comment (zainafzal08) [reply backend/app.py]: yep

File: backend/templates/index.html (hunk 2 above)
Comment (zainafzal08) [reply backend/templates/index.html]: was originally, made redundant by swagger

File: backend/requirements.txt (hunk 3 above)
Comment (zainafzal08) [reply backend/requirements.txt]: no, removed.

**Pull request merged**